src/features/contentCreator/initPage.ts

```diff
@@ -422,10 +422,11 @@ export class AnsibleCreatorInit {
   }
 
   public async openLogFile(fileUrl: string) {
-    vscode.commands.executeCommand(
-      "vscode.open",
-      vscode.Uri.parse(`vscode://file/${fileUrl}`)
-    );
+    const logFileUrl = vscode.Uri.file(fileUrl).fsPath;
+    console.log(`[ansible-creator] New Log file url: ${logFileUrl}`);
+    const parsedUrl = vscode.Uri.parse(`vscode://file${logFileUrl}`);
```

Comment:
@priyamsahoo Does this correctly work for MacOS?

Reply:
Yes. Tested it on the Mac as well. Seems like Mac has a better URI handling mechanism :)